test/serializable_resource_test.rb

```diff
@@ -2,8 +2,13 @@
 
 module ActiveModel
   class SerializableResourceTest < Minitest::Test
+    class Profile < ::Model
```

Comment:
Note that `::Model` is needed (instead of `Model`) because it would resolve to `ActiveModel::Model`.